hwdb.d/70-mouse.hwdb

```diff
@@ -190,6 +190,16 @@ mouse:usb:v413cp301a:name:PixArt Dell MS116 USB Optical Mouse:*
 mouse:usb:v0461p4d46:name:USB Optical Mouse:*
  MOUSE_DPI=1000@125
 
+##########################################
+# Elecom
+#########################################
+
+# Elecom HUGE TrackBall (M-HT1DR)
+mouse:usb:v056ep010d:name:ELECOM TrackBall Mouse HUGE TrackBall:*
+ MOUSE_DPI=500@125 *1000@125 1500@125
+ MOUSE_WHEEL_CLICK_ANGLE=15
+ MOUSE_WHEEL_CLICK_COUNT=24
```

Comment:
sorry for the late comment, missed this in the github view at first: these lines aren't necessary, they are the default values anyway and it's better to skip them to reduce noise.

Reply:
Do you reckon it might be good for the default value to be checked for in the scripts, so we don't add default values by mistake?

Reply:
Removed these lines, but I'll leave this unresolved so @hadess can get an answer to their question.

Reply:
> Do you reckon it might be good for the default value to be checked for in the scripts, so we don't add default values by mistake?

we could do this, but ... I'm too lazy to figure out the test here. There's a dependency of the properties, e.g. having the `_HORIZONTAL` one requires the normal one and `CLICK_COUNT` requires `CLICK_ANGLE`. That leaves us with a tree of "the default value should be specified when this and that value is present" and, given that this is the first time in years someone's added the default *and* there's no actual downside to adding the default angles/counts, it's not worth the effort IMO.